pkg/registry/apis/secret/inline/inline_secure_value.go

```diff
@@ -198,14 +199,18 @@ func (s *LocalInlineSecureValueService) CreateInline(ctx context.Context, owner
 	// TODO(2025-07-31): when we migrate to using the common type, we don't need this conversion.
 	secret := secretv1beta1.ExposedSecureValue(value)
 
-	decrypters := []string{serviceIdentity}
-	if owner.APIGroup != serviceIdentity {
-		decrypters = append(decrypters, owner.APIGroup)
+	// The owner group can always decrypt
+	decrypters := []string{owner.APIGroup, "??? UNIFIED STORAGE IDENTITY ???"}
+
+	serviceIdentity, ok := authInfo.GetExtra()[authn.ServiceIdentityKey]
```

Comment:
Although this is a slice, it should only contain 1 item (it's more of a quirk of the `GetExtra()`)